### ISSUE TITLE
Openapi v3 capabilities

### ISSUE THREAD

**Issue by kettesMark** [VanMoof/gopenapi#2]
Hi, This project is a really good initiative! I've looked into the code, but not too deep yet, and I saw traces of that it may support the usage of anyOf, allOf, oneOf. Could you, please provide an insight on how they should be use? I'm particularly interested in anyOf. Thank you very much!

**Comment by zetxek** [comment]
Hi @kettesMark, thanks for the comment in the repo! 
Despite it being hosted by the VanMoof organisation - we're not actively maintaining it right now. But it's open source anyway - so feel free to fork it and "adopt it".

And I'll also give @noamt a heads up as he was the creator of the repo and even if he's not at VanMoof anymore it's nice to get the attention on what you made :-)

**Comment by noamt** [comment]
Yes I actually haven't looked at this project for a while, but I will!
